### PR TITLE
Remove 'authors' from BOTMETA.yml

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Remove 'authors' from BOTMETA.yml

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`.github/BOTMETA.yml`

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -216,1072 +213,850 @@ files:
...
+    maintainers: skinp
   $modules/packaging/os/pacman.py:
-    authors: elasticdog indrajitr tchernomax
+    maintainers: elasticdog indrajitr tchernomax
     labels: pacman
     ignore: elasticdog
   $modules/packaging/os/pkgin.py:
-    authors: L2G jasperla szinck
-    maintainers: $team_solaris
+    maintainers: $team_solaris  L2G jasperla szinck
```
Comment (felixfontein) [inline .github/BOTMETA.yml:711]:
```suggestion
    maintainers: $team_solaris L2G jasperla szinck
```

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -216,1072 +213,850 @@ files:
...
   $modules/cloud/docker/docker_stack_task_info.py:
-    authors: imjoseangel
-  $modules/cloud/docker/docker_swarm.py:
-    authors: WojciechowskiPiotr tbouvet
-  $modules/cloud/docker/docker_swarm_info.py:
-    authors: WojciechowskiPiotr
+    maintainers: imjoseangel
   $modules/cloud/docker/docker_swarm_service.py:
-    authors: dariko hannseman jwitko
+    maintainers: dariko hannseman jwitko
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply .github/BOTMETA.yml]: and jwitko too..:) thanks! fixed

File: .github/BOTMETA.yml (hunk 1 above)
Comment (Andersson007) [reply .github/BOTMETA.yml]: good eye, thanks! filxed

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by felixfontein** [comment]
@Andersson007 thanks for fixing this!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Pull request merged**